docs/user/security/authentication/index.asciidoc

```diff
@@ -223,18 +281,48 @@ As with the previous SSOs, make sure that you have configured {es} first accordi
 
 [source,yaml]
 -----------------------------------------------
-xpack.security.authc.providers: [kerberos, basic]
+xpack.security.authc.providers:
+  kerberos.kerberos1:
+    order: 0
+    description: "Log in with Kerberos"
+  basic.basic1:
+    order: 1
 -----------------------------------------------
 
-As a reminder, the order is important as it determines the order in which each authentication provider is attempted.
-
 Kibana uses SPNEGO, which wraps the Kerberos protocol for use with HTTP, extending it to web applications. At the end of the Kerberos handshake, Kibana will forward the service ticket to Elasticsearch. Elasticsearch will unpack it and it will respond with an access and refresh token which are then used for subsequent authentication.
+
+[[http-authentication]]
+==== HTTP authentication
+
+[IMPORTANT]
+============================================================================
+Be very careful when you modify HTTP authentication settings as it may indirectly affect other important {kib} features that implicitly rely on HTTP authentication (e.g. Reporting).
+============================================================================
+
+HTTP protocol provides a simple authentication framework that can be used by a client to provide authentication information. It uses a case-insensitive token as a means to identify the authentication scheme, followed by additional information necessary for achieving authentication via that scheme.
+
+This type of authentication is usually useful for machine-to-machine interaction that requires authentication and where human intervention is not desired or just infeasible. There are a number of use cases when HTTP authentication support comes in handy for {kib} users as well.
+
+By default {kib} supports <<api-keys, `ApiKey`>> authentication scheme _and_ any scheme supported by the currently enabled authentication provider. For example, `Basic` authentication scheme is automatically supported when basic authentication provider is enabled, or `Bearer` scheme when any of the token based authentication providers is enabled (Token, SAML, OpenID Connect, PKI or Kerberos). But it's also possible to add support for any other authentication scheme in the `kibana.yml` configuration file, as follows:
+
+NOTE: Don't forget to explicitly specify default `apikey` scheme when you just want to add a new one to the list.
+
+[source,yaml]
+--------------------------------------------------------------------------------
+xpack.security.authc.http.schemes: [apikey, basic, something-custom]
+--------------------------------------------------------------------------------
+
+With the configuration above users can send requests to {kib} with the `Authorization` header using `ApiKey`, `Basic` or `Something-Custom` HTTP schemes (case insensitive). Under the hood {kib} will just relay this header to {es} that will try to authenticate request using the credentials provided within the header.
```

Comment:
To meet accessibility standards, we are removing instances of `above` from the docs. I would change this to:

`With this configuration, you can send requests to...`